compiler/rustc_middle/src/lib.rs

```diff
@@ -51,7 +51,6 @@
 #![feature(negative_impls)]
 #![feature(never_type)]
 #![feature(ptr_alignment_type)]
-#![feature(round_char_boundary)]
```

Comment:
Won't you need to `cfg_attr(bootstrap, …)` the feature in the compiler itself (two instances)?

Reply:
You'll need to do this in Clippy's `clippy_lints` crate as well, since it can now be built by the stage0 compiler.